Formula/mame.rb

```diff
@@ -76,15 +76,18 @@ def install
                    "USE_SYSTEM_LIB_PUGIXML=1",
                    "USE_SYSTEM_LIB_RAPIDJSON=1",
                    "USE_SYSTEM_LIB_SQLITE3=1",
-                   "USE_SYSTEM_LIB_UTF8PROC=1"
+                   "USE_SYSTEM_LIB_UTF8PROC=1",
+                   "TOOLS=1"
     bin.install "mame"
+    bin.install "chdman"
```

Comment:
This is provided via `rom-tools`.

Reply:
Didn't know about that! I'll cancel this PR, thanks for the info.